Uplift/Supporting/Constants.swift

```diff
@@ -65,6 +65,7 @@ struct Identifiers {
     static let habitTrackerOnboardingCell = "habitTrackerOnboardingCell"
     static let lookingForCell = "lookingForCell"
     static let noHabitsCell = "noHabitsCell"
+    static let facilitiesPriceInformationCell = "facilitiesPriceInformationCell"
```

Comment:
can we move this to be with the identifiers for the other facility related cells?